src/config/web3/utils/index.ts

```diff
@@ -361,3 +361,17 @@ export const handleCheckProviderChainId = async (provider: any): Promise<boolean
     return await setupNetwork({ network: selectedChainId, provider });
   }
 };
+
+export const addressToBytes32 = (address: string): string => {
+  if (!address.startsWith('0x')) {
+    throw Error('Address must start with 0x');
+  }
+  const abiCoder = new ethers.utils.AbiCoder();
+  const byte32Address = abiCoder.encode(['address'], [address]);
+
+  if (byte32Address.length !== 66) {
```

Comment:
Please create a constant with descriptive name

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1267/commits/615ba74ea46a24cbe4ce715707d21c7804c5f487) commit.